catalogue/webapp/pages/concept.tsx

```diff
@@ -173,7 +184,9 @@ export const ConceptPage: NextPage<Props> = ({
                     text: (
                       <>
                         {`By this ${conceptResponse.type.toLowerCase()} `}
-                        <span className="is-hidden-s">{`(${imagesBy.totalResults})`}</span>
+                        <span className="is-hidden-s">{`(${formatNumber(
+                          imagesBy.totalResults
+                        )})`}</span>
```

Comment:
The splitting of the formatting over multiple lines looks visually icky to me, although I’m not sure I can see a better way to fix it.

But since we do this several times, maybe pull this out into a component?

```typescript
type TabLabelProps = {
  text: string;
  totalResults: number;
}

const TabLabel: FunctionComponent<TabLabelProps> = ({ text, totalResults }) => (
  <>
    {text}
    <span className="is-hidden-s">({formatNumber(totalResults)})</span>
  </>
);
```